person.rb

```diff
@@ -0,0 +1,33 @@
+require_relative './nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
+  def initialize(name, age, parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] Similarly, let's update this method based on the comment on the one in the Book class.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```